src/app/components/Promo/a.jsx

```diff
@@ -1,12 +1,18 @@
 import styled from '@emotion/styled';
-import { C_EBON } from '@bbc/psammead-styles/colours';
+import { C_GREY_6, C_GREY_10 } from '@bbc/psammead-styles/colours';
 
 const A = styled.a`
-  color: ${C_EBON};
+  color: ${C_GREY_10};
   text-decoration: none;
   &:hover {
```

Comment:
no biggie but I think you can do

```suggestion
  &:hover, &:focus {
```

and remove the &:focus style below